classes/PodsI18n.php

```diff
@@ -0,0 +1,172 @@
+<?php 
+/**
+ * @package Pods
+ * @since 2.7
+ */
+class PodsI18n {
+
+	/**
+	 * @var PodsI18n
+	 */
+	static $instance = null;
+
+	/**
+	 * The strings
+	 */
+	public $strings = array();
+
+	/**
+	 * The localized strings
+	 */
+	public $localized = array();
+
+	/**
+	 * Singleton handling for a basic pods_i18n() request
+	 *
+	 * @return \PodsI18n
+	 *
+	 * @since 2.7
+	 */
+	public static function init() {
+
+		if ( ! is_object( self::$instance ) ) {
+			self::$instance = new PodsI18n();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Constructur
+	 */
+	public function __construct() {
+		//self::init();
+		$this->localize_assets();
+	}
+
+	/**
+	 * Localize assets
+	 * 
+	 * @since 2.7
+	 */
+	public function localize_assets() {
+		// Create existing strings of this class
+		$this->strings = $this->create_strings();
+
+		/**
+		 * Add strings to the localization
+		 * Setting the key of your string to the original (non translated) value is mandatory
+		 * Note: Existing keys in this class will overwrite the ones of this filter!
+		 * 
+		 * @since 2.7
+		 * @see create_strings()
+		 * @param array
+		 * @return array format: 'Untranslated string' => 'Translated string with use of WP translate functions'
+		 */
+		$strings_extra = apply_filters( 'pods_localized_strings', array() );
+
+		$this->strings = array_merge( $strings_extra, $this->strings );
```

Comment:
array_merge should overwrite our keys with those coming from the filter.
I'd say we want the keys added with this class to be forced as defaults.